e2e/process-services-cloud/edit-process-filters-component.e2e.ts

```diff
@@ -81,10 +81,6 @@ describe('Edit process filters cloud', () => {
         await editProcessFilter.checkSaveButtonIsDisplayed();
         await editProcessFilter.checkSaveAsButtonIsDisplayed();
         await editProcessFilter.checkDeleteButtonIsDisplayed();
-        await expect(await editProcessFilter.checkSaveButtonIsEnabled()).toEqual(false);
-        await expect(await editProcessFilter.checkSaveAsButtonIsEnabled()).toEqual(false);
-        await expect(await editProcessFilter.checkDeleteButtonIsEnabled()).toEqual(false);
```

Comment:
is it necessary to remove those checks that check if the buttons are enabled/disabled? Also should we leave the test without any expectation? I know that those methods check the visibility of the button and if it is not there it will fail but if let's say someone refactors the method `checkDeleteButtonIsDisplayed ` in the future to not fail but to catch the error and return a value true/fails. Could it lead to an ever-green test

Reply:
My concern is that the test is already doing lots of stuff that is not in the test name... if someone changes something in the future, that someone will need to write a proper test? :)